docs/Using Fleet/MDM-migration-guide.md

```diff
@@ -7,45 +7,169 @@ This section provides instructions for migrating your hosts away from your old M
+The end user migration workflow is supported for automatically enrolled (DEP) hosts.
+
+To watch a GIF that walks through the end user experience for the end user migration workflow, in the Fleet UI, head to **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+In Fleet, you can configure the end user workflow using the Fleet UI or fleetctl command-line tool.
+
+Fleet UI:
+
+1. Select the avatar on the right side of the top navigation and select **Settings > Integrations > Mobile device management (MDM)**.
 
-The migration worfklow is supported for automatically enrolled (DEP) hosts.
+2. Scroll down to the **End user migration workflow** section and select the toggle to enable the workflow.
 
-During the end user migration workflow, an end user's device will have their selected system
-theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds,
-you can optionally set an alternate logo for the themes.
+3. Under **Mode** choose a mode and enter the webhook URL for you automation tool (ex. Tines) under **Webhook URL** and select **Save**.
 
-You can do this in the Fleet UI by going to **Settings** > **Organization settings** >
-**Organization info** and adding a url to the desired image in the **Organization avatar URL (for
-dark backgrounds)** and **Organization avatar URL (for light backgrounds)** inputs. The appropriate
-image will show depending on the selected system theme.
+4. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
+
+    1. Head to **Settings** > **Organization settings** >
+**Organization info** and add URLs to your logos in the **Organization avatar URL(for dark backgrounds)** and **Organization avatar URL (for light backgrounds)** fields.
+    
+    2. Select **Save**.
+
+fleetctl CLI:
+
+1. Create `fleet-config.yaml` file or add to your existing `config` YAML file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_migration:
+      enable: true
+      mode: "voluntary"
+      webhook_url: "https://example.com"
+  ...
+```
+
+2. Fill in the above keys under the `mdm.macos_migration` key. 
+
+To learn about each option, in the Fleet UI, select the avatar on the right side of the top navigation, select **Settings > Integrations > Mobile device management (MDM)**, and scroll down to the **End user migration workflow** section.
+
+3. During the end user migration workflow, an end user's device will have their selected system theme (light or dark) applied. If your logo does not look good on either light or dark backgrounds, you can optionally set a logo for each theme:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  org_info:
+    org_logo_url: https://fleetdm.com/images/press-kit/fleet-blue-logo.png
```

Comment:
Wait, so this is the default logo AND it's the black theme logo? And only if you set ligght_background logo below, then you get the light logo? Is that right? It does not seem ocnsistent with the text in bullet 3.

Reply:
Another way to ask this is, why isn't there three logo options?
1. org_logo_url
2. org_logo_light_background_url
3. org_logo_dark_background_url

Reply:
> default logo AND it's the black theme logo?

Not quite. The Fleet logo is the default for `org_logo_url` and `org_logo_light_background_url `

If the user updates `org_logo_url` this only updates the logo in the Fleet top navigation and black theme:

![Screenshot 2023-08-10 at 3 26 03 PM](https://github.com/fleetdm/fleet/assets/47070608/7e8de57b-02a6-4c1d-8901-31891b9ba2dd)

If the user updates `org_logo_light_background_url `, this only updates the logo in the light theme. The Fleet logo is still used in top nav and dark theme.

I updated the docs to clarify this.

Reply:
I think it needs sentence here that's like, "if you only set `org_logo_url`, then it will be used for both dark and light mode"

Reply:
This isn't the behavior. Good feedback tho. I updated the docs to further clarify:

"Add URLs to your logos that look good on a dark background and light background in the `org_logo_url` and `org_logo_url_light_background` keys respectively. If you only set a logo for one, the Fleet logo will be used for the other."